src/test/java/com/google/sps/integration/ElectionListIT.java

```diff
@@ -66,15 +66,13 @@ public void electionSelection_onClick_redirectAndUpdateElectionQueryParameters()
     WebElement learnMoreButton = driver.findElement(By.id("state-learn-more-button"));
     learnMoreButton.click();
 
-    String targetElectionName = "Georgia+General+Primary+Runoff+Election";
     String targetElectionID = "4979";
 
     Thread.sleep(2000);
 
     Assert.assertEquals(
         String.format(
-            "http://localhost:9876/electionInfo.html?state=ga&electionId=%s&electionName=%s",
-            targetElectionID, targetElectionName),
+            "http://localhost:9876/electionInfo.html?state=ga&electionId=%s", targetElectionID),
```

Comment:
looks like this test is still failing.

Reply:
this has to do with the integration.db -- I'm going to write a script for it so that it automatically updates each time the test runs